tests/admin/applications/test_index.py

```diff
@@ -0,0 +1,79 @@
+class TestAdminElements:
+
+    def test_homepage_elements(self, app, client):
+        with app.app_context():
+            with client.session_transaction() as session:
+                session['signedIn'] = 'test.email@example.com'
+
+            response = client.get('/applications')
+            html = response.data.decode()
+            assert 'Download GRC applications' in html
+
+    def test_new_applications_present(self, app, client, submitted_application):
+        with app.app_context():
+            with client.session_transaction() as session:
+                session['signedIn'] = 'test.email@example.com'
+
+            print('Firing up with:', submitted_application.application_data())
+            print('Application created with reference number:', submitted_application.application_data().reference_number)
+
+            response = client.get('/applications#new')
+            html = response.data.decode()
+            assert 'New applications' in html
+            assert 'View application' in html
+            assert 'Reference number' in html
+            assert 'Applicant name' in html
+            assert 'Submitted' in html
+            assert '/applications/ABCD1234' in html
+            assert 'ABCD1234' in html
+            assert f'{submitted_application.user_input}' in html
+            assert f'{submitted_application.completed.strftime("%d/%m/%Y %H:%M")}' in html
```

Comment:
Can you hard code this as well. Set the submission datetime in the fixture or in the test and then assert the value here.